AutomaticInterface/Tests/AccessibilityTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Linq;
+using AutomaticInterface;
+using FluentAssertions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Xunit;
+
+namespace Tests;
+
+public partial class AccessibilityTests
+{
+    private static string GenerateCode(string code)
```

Comment:
Hm, this dublicates code in the original tests.
Maybe use the same partial class tests I've used like in AutomaticInterface/Tests/GeneratorsTests.MethodParameters.cs?